models.html

```diff
@@ -0,0 +1,134 @@
+            1 x 10-minute pose
+            [BREAK]
+            2 x 10-minute poses
+            [BREAK]
+            1 x 20-minute pose
+            [BREAK]
+            1 x 20-minute pose
+            [AFTER PARTY]
+            .</div>
+
+        <div class="question">What should I bring with me?</div>
+        <div class="answer">Robe or cover-up (before taking the posing stand, during breaks, and for the after-party), wet wipes (for freshening up), props (if discussed with the organizer), and something to drink (wine is encouraged).</div>
+
+        <div class="question">What should I do to prepare?</div>
+        <div class="answer">Practice at home. Stand in front of a mirror, grab a timer, and gauge how it feels to hold your body in different positions for varying lengths of time. Share your socials with the organizer if you seek to grow your media empire.</div>
+
+        <div class="question">What about personal grooming?</div>
+        <div class="answer">Arrive clean, but skip the body moisturizer (you don’t want to slip while holding a pose). With regard to hair removal, this seemingly personal practice has, in fact, profoundly influenced science, medicine, business, and even warfare–prompting enormous outlays of money, emotion, and environmental impact. Consider your personal values and comfort when making such decisions.</div>
+
+        <div class="question">How should I pose?</div>
+        <div class="answer">TL;DR your first poses should be full of movement. Face a different direction for each new pose. Additional guidance below.</div>
+
+        <div class="question">Where can I find inspiration for my poses?</div>
+        <div class="answer">Through your posture, expression, and interaction with space, you can convey narratives, emotions, and character, transforming a static pose into a dynamic scene. You may find it helpful to mine inspiration from the following sources: 1) your body/features, 2) art/fashion, 3) sports/movement practices, and 4) sensuality/kink.</div>
+
+        <div class="question">What should I do during breaks?</div>
+        <div class="answer">Prioritize your own comfort. If you need to rest, stretch, hydrate, or use the restroom, do that. Feel free to wander around the room to admire everyone’s hard work and/or make new friends.</div>
+
+        <div class="question">Do you have any final modeling tips?</div>
+        <div class="answer">Take a couple seconds to find each pose. Trust the organizer to start the timer when it looks like you’ve “settled” in. Be still. For the longer poses (e.g., 20 minutes), remember that you’re only human, and it’s ok to adjust and return to a pose if needed.</div>
+
+        <div class="question">What are the compositional elements of a pose?</div>
+        <div class="answer"> You are using your body to create shapes and lines, and to communicate expression. Here are 8 basic principles to keep in mind while creating those shapes and lines.    
```

Comment:
To make a numbered list use HTML, https://www.w3schools.com/html/html_lists_ordered.asp.